newswires/client/src/App.tsx

```diff
@@ -174,215 +173,26 @@ export function App() {
-
-								<EuiHeaderSection side={'right'}>
-									<EuiHeaderSectionItem>
-										<EuiFlexGroup
-											gutterSize="xs"
-											css={css`
-												margin-left: 8px;
-											`}
-										>
-											<EuiShowFor sizes={['xs', 's']}>
-												<Tooltip
-													tooltipContent={'Open new ticker'}
-													position="left"
-												>
-													<EuiButtonIcon
-														aria-label="New ticker"
-														display="base"
-														size="s"
-														iconType={'popout'}
-														onClick={() => openTicker(config.query)}
-													/>
-												</Tooltip>
-											</EuiShowFor>
-											<EuiShowFor sizes={['m', 'l', 'xl']}>
-												<EuiButton
-													size="s"
-													iconType={'popout'}
-													onClick={() => openTicker(config.query)}
-												>
-													New ticker
-												</EuiButton>
-											</EuiShowFor>
-											<SettingsMenu />
-										</EuiFlexGroup>
-									</EuiHeaderSectionItem>
-								</EuiHeaderSection>
-							</EuiHeader>
-						)}
-						{isPoppedOut && (
-							<h1
-								css={css`
-									display: none;
-								`}
-								className="sr-only"
-							>
-								Newswires
-							</h1>
-						)}
-						{status !== 'error' && (
-							<>
-								<EuiShowFor sizes={['xs', 's']}>
-									{isPoppedOut && (
-										<ResizableContainer
-											Feed={Feed}
-											Item={
-												selectedItemId ? (
-													<ItemData id={selectedItemId} />
-												) : undefined
-											}
-											directionOverride={'vertical'}
-											setSideNavIsOpen={setSideNavIsOpen}
-										/>
-									)}
-
-									{view === 'item' && !isPoppedOut && (
-										<ItemData id={selectedItemId} />
-									)}
-
-									{view !== 'item' && !isPoppedOut && (
-										<Feed setSideNavIsOpen={setSideNavIsOpen} />
-									)}
-								</EuiShowFor>
-								<EuiShowFor sizes={['m', 'l', 'xl']}>
-									<ResizableContainer
-										Feed={Feed}
-										Item={
-											selectedItemId ? (
-												<ItemData id={selectedItemId} />
-											) : undefined
-										}
-										setSideNavIsOpen={setSideNavIsOpen}
-									/>
-								</EuiShowFor>
-							</>
-						)}
-						{status == 'error' && (
-							<EuiEmptyPrompt
-								css={css`
-									background: white;
-								`}
-								actions={[
-									<EuiButton
-										onClick={handleRetry}
-										key="retry"
-										iconType={'refresh'}
-									>
-										Retry
-									</EuiButton>,
-									<EuiButton
-										onClick={() => handleEnterQuery(defaultQuery)}
-										key="clear"
-										iconType={'cross'}
-									>
-										Clear
-									</EuiButton>,
-								]}
-								body={<p>Sorry, failed to load because of {state.error}</p>}
-								hasBorder={true}
-							/>
-						)}
-					</div>
+					{!!config.ticker && (
+						<TickerLayout errorPromptComponent={errorPromptComponent} />
```

Comment:
I'm curious why the component is being passed in as an explicit prop, rather than making the error component a child of the parent?
```
<TickerLayout>
 { status === 'error' && <ErrorComponent/> }
</TickerLayout>
```